Classes/ARLocalTiledImageDataSource.m

```diff
@@ -11,16 +11,51 @@
 
 @implementation ARLocalTiledImageDataSource
 
+- (id)init
+{
+    self = [super init];
+    if (self) {
```

Comment:
I don't think this should be initialized to anything, I would leave it `nil` and handle that.